crates/rolldown_common/src/chunk/mod.rs

```diff
@@ -255,8 +255,13 @@ impl Chunk {
         hash
       }
     });
+    let chunk_name = if options.preserve_modules {
+      self.get_preserve_modules_chunk_name(options, chunk_name.as_str())
+    } else {
+      Cow::Borrowed(chunk_name.as_str())
+    };
```

Comment:
https://github.com/rolldown/rolldown/blob/d36407c1f591805fc98d55a35a70a0662241c151/crates/rolldown_common/src/chunk/mod.rs?plain=1#L188-L192, this may be extracted as a function?